Stripe/STPPaymentMethodsViewController.m

```diff
@@ -266,41 +300,3 @@ - (void)dismissWithCompletion:(STPVoidBlock)completion {
 }
 
 @end
-
-@implementation STPPaymentMethodsViewController (Private)
```

Comment:
had to move this out of the category to get things to compile, also not sure why